fuzzers/072-site_pins/generate.tcl

```diff
@@ -0,0 +1,69 @@
+# This script first finds a list of all tile types.  It then outputs all nodes
+# within each tile, and any connected site_pins.  It also outputs all sites
+# present within the tile, its site type, and the list of site pins.
+#
+# This information should be sufficent to map the tile routing resources to
+# the sites within a tile.
+#
+# The output format is JSON5, which can be converted to JSON with json5.tool:
+# python -mjson5.tool --json {input JSON5 file} > {output JSON file}.
```

Comment:
We've standardized on JSON. Can we add that to the Makefile?

Reply:
Done